server_health_check-rails.gemspec

```diff
@@ -21,8 +21,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "rails", ">= 3.0", "< 6.0"
-  spec.add_runtime_dependency "server_health_check", "~> 1.0"
+  spec.add_runtime_dependency "rails", "~> 5.0"
```

Comment:
My previous comments I think makes this irrelevant. But even so, I'm not aware of Rails `3.*` or `4.*` being limited to Ruby less than 2.3....